common/endpoints/setup.go

```diff
@@ -0,0 +1,51 @@
+package endpoints
+
+import (
+	"log"
+	"time"
+
+	"github.com/scootdev/scoot/common/stats"
+	"github.com/scootdev/scoot/config/jsonconfig"
+	"github.com/scootdev/scoot/ice"
+)
+
+// Module returns a module that supports serving HTTP
+// Important to also install a function that returns map[string]http.Handler
+func Module() ice.Module {
+	return module{}
+}
+
+type module struct{}
+
+// Install installs functions for serving HTTP
+func (m module) Install(b *ice.MagicBag) {
+	b.PutMany(
+		func(scope StatScope) stats.StatsReceiver {
+			return MakeStatsReceiver(scope).Precision(time.Millisecond)
+		},
+		NewTwitterServer,
```

Comment:
I agree in general; I'd say ObservableServer?

In the interest of not getting hung up on naming, especially given how little this is actually touching that function, I've TODO'ed it. That work for you?